src/metabase/api/search.clj

```diff
@@ -24,7 +24,8 @@
             [metabase.util.honeysql-extensions :as hx]
             [metabase.util.schema :as su]
             [schema.core :as s]
-            [toucan.db :as db]))
+            [toucan.db :as db]
```

Comment:
looks like `toucan.db` isn't necessary here?

Reply:
We use `db/reducible-query`

Reply:
ah. i didn't see it in the diff and missed it was in this diff because of the closing parens. sorry about that.